### PR TITLE
corrected PhpDocs

### PR DESCRIPTION
**Pull request by jasonw4331**
created .gitignore
added event priorities

### CODE REVIEW

File: src/BoxOfDevs/CommandUsage/RegisterTask.php [hunk 1]
```diff
@@ -13,17 +13,20 @@
...
-	@param     $tick    int
-	*/
+	public function __construct(Main $owner) {
+		parent::__construct($owner);
+	}
+
+  /**
+   * Runs when the tasks runs
+   * @param int $tick
+   */
```
Comment (jasonw4331) [inline src/BoxOfDevs/CommandUsage/RegisterTask.php]: oops. I need to fix this formatting

Comment (AvgZing) [reply src/BoxOfDevs/CommandUsage/RegisterTask.php]: XD

**Comment by AvgZing** [comment]
Once the formatting is fixed, and this is tested, then it will be merged.

**Comment by AvgZing** [comment]
Now you just have to test it then tell us

**Comment by jasonw4331** [comment]
formatting and PhpDoc changes do not affect the build

**Comment by AvgZing** [comment]
typo is possible. test please. Policy is not to be interfered with

**Comment by lavender-sys** [comment]
Since when did we have PhpDoc?

**Comment by AvgZing** [comment]
@Dog2puppy We don't...

**Comment by jasonw4331** [comment]
PhpDocs were just a comment system  to help IDE's recognise what variables are or could be before strict types were introduced with php7

**Comment by ghost** [comment]
C r i n g e

**Pull request closed without merging**